src/control-center/src/ControlCenter.cpp

```diff
@@ -6,13 +6,18 @@
 #include <blocks/BlockReader.h>
 #include <blocks/RequestConfigBlock.h>
 
-ControlCenter::ControlCenter(Serializer serializer) : serializer(serializer) {
+ControlCenter::ControlCenter(Serializer serializer, const std::string& filepath) : serializer(serializer) {
+    cfg = new ConfigReader(filepath);
```

Comment:
`ControlCenter` wczytujące swój config z pliku brzmi trochę, jak odwrócenie zależności. Sądzę, że `ControlCenter` powinno być tworzone z konfiguracją (np. wczytaną w `main()`) przekazaną jako np. struktura. Ułatwia to dodatkowo testy (można łatwo tworzyć `ControlCenter` z różnymi konfiguracjami).